src/src/PluginStructs/P145_data_struct.cpp

```diff
@@ -11,14 +11,41 @@
 #include "../Globals/ESPEasyWiFiEvent.h"   // Need to know when WiFi is ruining the ADC measurements
 #include "../Helpers/Hardware.h"           // Need to know the ADC properties
 
+// The table sensorDefs[] contains string items for representation. 
+// Storage is in PROGMEM where a (fixed format) C-style string does not fit well
+// An enum like int is used instead, together with a enum to string conversion function
+// Below the defines used to represent these enums
+
+// Gas type identifiers used to condense the sensorDefs table to fixed format
+#define GAS_USER    0
+#define GAS_CO2     1
```

Comment:
These #defines should have a `P145_` prefix, to avoid possible clashes with other (future) plugins.